### ISSUE TITLE
Don't work with grav latest version

### ISSUE THREAD

**Issue by ossplus** [gautamkrishnar/grav-openshift-quickstart#1]
Grav have decided to change the minimum requirements for Grav from PHP 5.4.40 to 5.5.9 to stay in line with the new minimum version of some of key vendor libraries.

so you should change to use openshift-diy-php

**Comment by gautamkrishnar** [comment]
Thanks @jojouser I Will modify the code soon

**Comment by rhcarvalho** [comment]
@gautamkrishnar you could update the quick start to use this cartridge here:
https://github.com/boekkooi/openshift-cartridge-php

And use PHP 5.6.16 :+1:

**Comment by gautamkrishnar** [comment]
Ok thanks a lot :+1:  @rhcarvalho

**Comment by msrnoob** [comment]
@gautamkrishnar please update the one click installer for the latest version of grav.

**Comment by gautamkrishnar** [comment]
@imosman sure...

**Comment by gautamkrishnar** [comment]
@rhcarvalho started working  on this issue on diy branch

**Comment by gautamkrishnar** [comment]
@laobubu i am using your code of "openshift-php5.5-cgi-apache" and i am getting a 404.. any idea on this? 
check out the branch:
https://github.com/gautamkrishnar/grav-openshift-quickstart/tree/diy

**Comment by laobubu** [comment]
After the first git-pushing to OpenShift, you shall run `bash ${OPENSHIFT_REPO_DIR}/misc/make.sh` via SSH, to compile the PHP 5.5.18. (that's what my `index.cgi` does)

You can also choose another php version by editing `misc/make.sh`. Good luck.

BTW, it will be great if you update README by adding a link to https://github.com/laobubu/openshift-php5.5-cgi-apache

**Comment by gautamkrishnar** [comment]
@laobubu thanks. Sure i will do it :smile:

**Comment by gautamkrishnar** [comment]
@laobubu calling it in the build script on action hooks dir, doesn't works. Is there any other way i can call it automaticially when the app is deployed?

**Comment by laobubu** [comment]
Merge #2 and solve :smile:

**Comment by gautamkrishnar** [comment]
@laobubu thanks a lot :+1:

**Comment by rhcarvalho** [comment]
:tada: nice

**Comment by gautamkrishnar** [comment]
Still showing 404 :(

**Comment by laobubu** [comment]
Seems you deleted .htaccess and /admin doesn't exist.
1. Modify `httpd.conf` , `AllowOverrid` from `None` to `All`
2. Your `.htaccess`

**Comment by gautamkrishnar** [comment]
@laobubu ok thanks. i will do it soon

**Comment by ghost** [comment]
Has /admin been fixed yet? I am unable to access it.

**Comment by gautamkrishnar** [comment]
yes sorry for that. @Shuttle099 currently i don't have much time to spend on this. You can still ssh into your server and run the following commands to install the admin plugin:

``` bash
cd app-root/repo/php/bin/bin/
gpm install admin
```

**Comment by gautamkrishnar** [comment]
@jojouser @hexplor @laobubu Can you help me on this please...

**Comment by gautamkrishnar** [comment]
@laobubu it is running PHP 5.3.3, but Grav needs at least PHP 5.5.9 to run...
How to set that version of PHP??

**Comment by laobubu** [comment]
sorry but not got your point.

**Comment by gautamkrishnar** [comment]
Oh sorry, just a typo, How to set the php version of the cartridge ? @laobubu

**Comment by laobubu** [comment]
Hard way: Create your own components, like
https://github.com/boekkooi/openshift-cartridge-php

Easy way: create a DIY cartridge. If you want to use your own PHP during
using SSH, please read:
https://github.com/laobubu/openshift-php5.5-cgi-apache/wiki/FAQ-(-have-problem%3F-read-this-first!-)#wrong-php-version-on-ssh--scripts

2016-04-16 16:27 GMT+08:00 Gautam krishna.R notifications@github.com:
